src/eradiate/experiments/_dem.py

```diff
@@ -119,6 +125,40 @@ class DEMExperiment(EarthObservationExperiment):
     def __attrs_post_init__(self):
         self._normalize_atmosphere()
         self._normalize_measures()
+        self._normalize_surface()
+
+    def _normalize_surface(self) -> None:
```

Comment:
You'll also have to check if the other normalization steps shouldn't take the DEM into account (*e.g* distant measure targeting and atmosphere geometry).

Reply:
We have to discuss this during the Eradiate meeting.